lib/modules/commentNavigator.js

```diff
@@ -89,7 +88,7 @@ modules['commentNavigator'] = {
 						case 'images':
 							thisEle.setAttribute('title', 'Navigate through comments with images');
 							break;
-						case 'images':
+						case 'videos':
```

Comment:
gulp copies the files to another folder, instead of symlinking.  so yes, you won't have issues with the symlinks =p